ecs-logs-collector.sh

```diff
@@ -584,6 +585,24 @@ enable_ecs_agent_debug() {
   esac
 }
 
+get_docker_goroutines() {
+  try "dump docker goroutine stacks"
+  dstdir="${info_system}/docker_stacks"
+  mkdir -p ${dstdir}
+  # send SIGUSR1 to dump goroutines to
+  # /var/run/docker/goroutine-stacks-<date>.log
+  if kill -SIGUSR1 $(pidof dockerd) ; then
+    # give docker time to write the file
+    sleep 5
+    # find file and copy to our output
+    if  find /var/run/docker/ -cmin -1 -type f -name "goroutine-stacks*.log" -exec cp -p '{}' $dstdir \; ; then
```

Comment:
This should be quoted too:

```suggestion
    if  find /var/run/docker/ -cmin -1 -type f -name "goroutine-stacks*.log" -exec cp -p '{}' "$dstdir/" \; ; then
```

Reply:
Added.